README.md

```diff
@@ -1 +1,191 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML, CSS</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Use of linters]**
+- **[Github & Git]**
+- **[README.md file]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [No Live Demo Required](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Node installed in computer
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd desired-folder
+  git https://github.com/Roathy/Hello---Microverse.git
+```
+
+### Install
+
+Install this project with:
+
+- No installation required
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html in browser
+
+### Run tests
+
+To run tests, run the following command:
+
+- No testing implement yet
+
+### Deployment
+
+You can deploy this project using:
+
+GitHub can be used to deploy website. For more information about publishing sources, see "About GitHub pages".
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@Roathy](https://github.com/Roathy)
+- Twitter: n/a
+- LinkedIn: [LinkedIn](https://linkedin.com/in/julio-vergara-roathy)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Desktop version]**
+- [ ] **[Details section]**
+- [ ] **[Responsiveness ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Roathy/Hello---Microverse/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Please remove this section from your `README.md` as of now there is nothing that a user might ask having the placeholders makes your `README.md` file unprofessional. Please read the below `NOTE` and keep remembering to also remove this section reference from the `table` which is at the beginning of this `README` file.

**📝 FACT: *FAQ* is a document (as on a website) that provides answers to a list of typical questions that users might ask regarding a particular subject**

**NOTE: Except FAQ section everything is required and important. So go ahead and add relevant things.**